scripts/react-native-xcode.sh

```diff
@@ -32,7 +32,7 @@ fi
 
 case "$CONFIGURATION" in
   *Debug*)
-    if [[ "$PLATFORM_NAME" == *simulator ]]; then
+    if [[ "$PLATFORM_NAME" == *simulator || "$PLATFORM_NAME" == macosx ]]; then
```

Comment:
Why does this need to change?

Reply:
It address Issue #334 and only impacts Debug builds.